modules/rostests/kmtests/ntos_ob/ObTypes.c

```diff
@@ -155,10 +157,24 @@ TestObjectTypes(VOID)
 {
     ULONG Index;
 
+    if (skip(g_NtVersion <= 0x0600, "TODO on NT6.1+: GetObjectType() fails\n"))
```

Comment:
Is `_WIN32_WINNT_VISTA` available here? (`sdkddkver.h`) if so, please use that.
Your comment mentions 6.1, but you actually check on 6.0 btw.

Reply:
Eventually, reverted.